src/core.cpp

```diff
@@ -57,6 +57,12 @@ namespace linuxdeploy {
                                std::string customAppRunPath, appdir::AppDir& appDir) {
         ldLog() << std::endl << "-- Deploying files into AppDir root directory --" << std::endl;
 
+        if (!customAppRunPath.empty()) {
+            ldLog() << LD_WARNING << "Deploying custom AppRun: " << customAppRunPath << std::endl;
```

Comment:
This should not be a warning, but a regular message.

There should be a warning if the file is overwriting an existing file, though.